suite/soak/tests/ocr_test.go

```diff
@@ -22,11 +27,17 @@ var _ = Describe("OCR Soak Test @soak-ocr", func() {
 
 	BeforeEach(func() {
 		By("Deploying the environment", func() {
-			env, err = environment.DeployOrLoadEnvironmentFromConfigFile(
-				"/root/test-env.json", // Default location for the soak-test-runner container
+			env = environment.New(
+				&environment.Config{InsideK8s: true, TTL: 12 * time.Hour},
```

Comment:
Assuming `TTL` will tear down the environment after time limit?

Reply:
Set default TTL to 168 hours for soak